tools/gyp/pylib/gyp/msvs_emulation.py

```diff
@@ -531,7 +531,7 @@ def GetCflags(self, config):
         cl("AdditionalOptions", prefix="")
         cl(
             "EnableEnhancedInstructionSet",
-            map={"1": "SSE", "2": "SSE2", "3": "AVX", "4": "IA32", "5": "AVX2"},
+            map={"1": "SSE", "2": "SSE2", "3": "AVX", "4": "IA32", "5": "AVX2", "6": "AVX512"},
```

Comment:
This should also be upstreamed to teh `gyp-next` repo. Can probably be in its own PR.

Reply:
Is this needed for the current PR?

I might be the one who added those, but if that is the case, I probably did so because libbase64 required some sophistication from the build engine. Yet libbase64 is gone now.

Reply:
Ok then we can ignore this change.